pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SchemasResourceBase.java

```diff
@@ -133,19 +134,51 @@ public void deleteSchema(boolean authoritative, AsyncResponse response) {
                 });
     }
 
-    public void postSchema(PostSchemaPayload payload, boolean authoritative, AsyncResponse response) {
-        validateDestinationAndAdminOperation(authoritative);
+    private CompletableFuture<SchemaCompatibilityStrategy> getSchemaCompatibilityStrategyAsync() {
```

Comment:
Does this `getSchemaCompatibilityStrategyAsync` have difference with `internalGetSchemaCompatibilityStrategy`?
Can share one method?